### PR TITLE
feat: Adds new inbox selector with more info for new message modal [cw-1358]

### PR DESCRIPTION
**Pull request by nithindavid**
## Description
Currently, we are using a normal dropdown which needs to be clearer as we need to show more information/type of inbox. Leading to confusion while choosing an inbox
image

Fixes - https://linear.app/chatwoot/issue/CW-1358/inbox-dropdown-component-to-use-across-product

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by nithindavid** [comment]
@iamsivin I have added the fix, Thanks for the review.

image

File: app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue [hunk 1]
```diff
@@ -129,10 +153,12 @@ import Thumbnail from 'dashboard/components/widgets/Thumbnail';
 import WootMessageEditor from 'dashboard/components/widgets/WootWriter/Editor';
 import ReplyEmailHead from 'dashboard/components/widgets/conversation/ReplyEmailHead';
 import CannedResponse from 'dashboard/components/widgets/conversation/CannedResponse.vue';
+import InboxDropdownItem from 'dashboard/components/widgets/InboxDropdownItem.vue';
```
Comment (iamsivin) [inline app/javascript/dashboard/routes/dashboard/conversation/contact/ConversationForm.vue:156]:
```suggestion
import InboxDropdownItem from 'dashboard/components/widgets/InboxDropdownItem';
```

**Comment by iamsivin** [comment]
@nithindavid I'm getting some error while clicking send message button. Can you take a look?
image

**Comment by nithindavid** [comment]
@iamsivin Thanks for pointing out the error. Please review again

**Comment by iamsivin** [comment]
> @iamsivin Thanks for pointing out the error. Please review again

@nithindavid I reviewed it and found the same issue with the WhatsApp cloud inbox, and the error message color has broken for the inbox. You can see it in the attached screenshot and screen recording.

image


https://user-images.githubusercontent.com/64252451/233146522-8a88f3ff-9fc4-4b9c-8c1e-5c5fb7270193.mov

**Review by iamsivin — APPROVED** [review]
LGTM 
https://www.loom.com/share/799ebfa10e284a609a2dab3cbe533b62

**Comment by iamsivin** [comment]
@scmmishra Can you also take a look?

**Pull request merged**